apps/server/src/modules/rocketchat-user/service/rocket-chat-user.service.ts

```diff
@@ -53,63 +53,95 @@ export class RocketChatUserService implements DeletionService, IEventHandler<Use
 		);
 		const rocketChatUser = await this.rocketChatUserRepo.findByUserId(userId);
 
-		if (rocketChatUser.length > 0) {
+		if (rocketChatUser !== null) {
```

Comment:
Please simplify this `if` statement to make it "less nested" - try to exit early, e.g. reverse this logic to not have this "non null" check at all, for example
```suggestion
		if (rocketChatUser === null) {
			const result = this.buildResultAndLog(0, 0, userId, 'RocketChat user already deleted', StatusModel.FINISHED);
  
			return result;
		}
```
and then put all the other code (without `else`).